Automattic/wp-calypso
# Onboarding import: update the outdated images

#### Proposed Changes

* Update the outdated images

#### Testing Instructions

* Head to `/setup/importerWordpress?siteSlug={SLUG}&from={DOMAIN}`
* The illustration underneath the text - `What would you like to import?` should be updated:

| before  |
|---|
| <img width="1091" alt="Screen Shot 2022-06-02 at 7 55 46 PM" src="https://user-images.githubusercontent.com/1024985/171787177-63a3ec95-2739-4396-a561-5d00b7185113.png"> |

| after |
| --- |
  | <img width="1135" alt="Screen Shot 2022-06-02 at 7 55 41 PM" src="https://user-images.githubusercontent.com/1024985/171787217-c7334e54-05b1-4a54-b131-005d7342dbed.png">  |

* Head to `/setup/importList?siteSlug={SLUG}`
* The illustration underneath the text - `Import your content from another platform` should be updated:

| before  |
|---|
| <img width="1132" alt="Screen Shot 2022-06-02 at 7 57 00 PM" src="https://user-images.githubusercontent.com/1024985/171787669-c2fe6f73-b22c-4d0f-8c4e-f7fc691cb96a.png"> |

| after |
| --- |
  | <img width="1110" alt="Screen Shot 2022-06-02 at 7 59 38 PM" src="https://user-images.githubusercontent.com/1024985/171787702-5863dfcb-39b2-4038-9853-954427b64217.png"> |

Related to https://github.com/Automattic/wp-calypso/issues/64279
